js/envato-market.js

```diff
@@ -236,6 +238,44 @@
       regex = new RegExp( '[\\?&]' + name + '=([^&#]*)' );
       results = regex.exec( location.search );
       return null === results ? '' : decodeURIComponent( results[1].replace( /\+/g, ' ' ) );
+    },
+
+    maybeLoadHelthcheck: function() {
```

Comment:
Typo: should this be `maybeLoadHealthcheck`?

Reply:
Yes, yes it should. Fixed! https://github.com/envato/wp-envato-market/pull/214/commits/989eacc937515bf41e5789bd1fe10ae444cb1788